config/local.sample.js

```diff
@@ -3,29 +3,31 @@ if (process.env.NODE_ENV !== 'test') {
     passport: {
       github: {
         options: {
-          clientID: '123abc',
-          clientSecret: '456def',
+          clientID: 'GITHUB_OAUTH_CLIENT_ID',
+          clientSecret: 'GITHUB_OAUTH_CLIENT_SECRET',
           callbackURL: 'http://localhost:1337/auth/github/callback',
         },
         organizations: [
-          1234567,
+          1234567, // YOUR GITHUB ORGANIZATION ID
         ],
       },
     },
     postgres: {
```

Comment:
We can move this out of `local.js` now that these are going to be constant, right?

Reply:
it still needs to be different from the test and production configs. would putting it in `config/postgres.js` do that?

Reply:
Yeah, it should. The environment configs should still override it there. Just make sure to drop the `postgres` key.